js/pages/data-sources/data-sources.js

```diff
@@ -105,14 +104,17 @@ define([
 			];
 
 			this.model = params.model;
-			this.sources = ko.computed(() => {
-				return sharedState.sources().filter(function (s) {
-					return s.hasResults && s.hasCDM;
-				});
+			this.sources = sharedState.sources().filter(function (s) {
```

Comment:
why did you make it not computed? I guess it will make you reload the page if you went on it unauthenticated and then logged in

Reply:
It also triggered a second reload when the report drop down was changed and was causing issue with the change to make the report loading driven by routes.  It seems safe to say that since the sources are loaded once at startup that computing the list of sources that have results or cdm with each reference to the variable was unnecessary.